files/file.go

```diff
@@ -254,19 +254,6 @@ func (i *FileInfo) detectType(modify, saveContent, readHeader bool) error {
 	case strings.HasPrefix(mimetype, "image"):
 		i.Type = "image"
 		resolution, err := calculateImageResolution(i.Fs, i.Path)
-
-        // Skip resolution calculation for TIFF files
-		ext := strings.ToLower(filepath.Ext(i.Path))
-		if ext == ".tif" || ext == ".tiff" {
-			return nil
-		}
-
-		if err != nil {
```

Comment:
Hi @zamuzakki 
I think this line is from upstream
We can't delete this as it is for the actual image type